src/libraries/Microsoft.VisualBasic.Core/tests/NewLateBindingTests.cs

```diff
@@ -96,6 +96,7 @@ public static IEnumerable<object[]> LateCall_OptionalValues_Data()
 
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsReflectionEmitSupported))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/51834", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/51834", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsAppleMobile))]
```

Comment:
Should we create a new issue for skipping this test on Apple platforms? The original issue is WASM specific.

Reply:
This shouldn't be WASM-specific. I would rather update the existing issue.

Reply:
Sounds good